### PR TITLE
Added Placeholder value for receiving participant when updating charges

### PR DESCRIPTION
**Pull request by blinblinblin**
#202 I added a placeholder value in the receiving participant item on the charge form page so that it does not make the users think the receiving_participant has disappeared when we update a charge. Could you verify if this is the desired behavior described in the issue tracker? @pkoenig10 Thanks!

### CODE REVIEW

**Comment by pkoenig10** [comment]
Yupp I can take a look later tonight.

File: .travis.yml [hunk 1]
```diff
@@ -5,3 +5,5 @@ bundler_args: --without development staging production
 sudo: false
 before_script:
   - bundle exec rake db:test:prepare
+
+
```
Comment (ChaseBro) [inline .travis.yml]: nit: extra blank line

File: test/unit/event_type_test.rb [hunk 2]
```diff
@@ -9,11 +9,12 @@
...
 #
 
 class EventTypeTest < ActiveSupport::TestCase
   # Relationships
   should have_many(:events)
   
 
-end
\ No newline at end of file
+end
```
Comment (ChaseBro) [inline test/unit/event_type_test.rb]: nit: extra blank line

**Comment by pkoenig10** [comment]
@blinblinblin Yeah I think this is a good enough solution to resolve the issue in #202.  Eventually it would be nice to have a better UI for entering users that allows us to nicely display participant fields rather than just use placeholder text.  But this is tricky given that those fields are generated from submitted Andrew IDs, swiped card numbers, or NFC card numbers.  I'll open another issue to track it.  Super low priority, but would be a welcome improvement.

**Comment by blinblinblin** [comment]
@pkoenig10 thanks for letting us know. Would you mind if I squash and merge it then?

**Pull request merged**